ubuntu_16.04.sh

```diff
@@ -20,17 +20,17 @@ export LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu
 export LIBRARY_PATH=/usr/lib/x86_64-linux-gnu
 echo "export LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu" >> ~/.bashrc
 cd $HOME
-mkdir opt
+mkdir -p opt
 cd opt
 # Install MOAB
-mkdir moab
+mkdir -p moab
 cd moab
 git clone https://bitbucket.org/fathomteam/moab
```

Comment:
I found a sample fix in the ubuntu script.  (Option 3 - ask the user what to do)

I'll merge this and PR a modular set of scripts that have this change proposed.